desktop/app/pro.go

```diff
@@ -109,44 +107,44 @@ func (app *App) IsProUserFast(uc common.UserConfig) (isPro bool, statusKnown boo
 // It loops forever in 10 seconds interval until the user is fetched or
 // created, as it's fundamental for the UI to work.
 func (app *App) servePro(channel ws.UIChannel) error {
-	chFetch := make(chan bool)
-	ctx := context.Background()
-	go func() {
```

Comment:
@atavism We were creating two users like in android, We don't need this anymore, Since we are creating user in afterStart

Reply:
Thanks for removing this!